packages/contracts/contracts/gatekeepers/freeForAll/FreeForAllCheckerFactory.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+import { Factory } from "@excubiae/contracts/proxy/Factory.sol";
```

Comment:
I think this should be made directly on Excubiae's repo, it's contracts that everyone else should be able to reuse by importing from Exucubiae repo. Same applies for the deploy scripts which we could just import from the npm package

Reply:
Yes, this is the idea. After I add all the logic here, I'll move it to excubiae repo and import here.
However, deploy script still be here because excubiae has different deploy logic.

Reply:
fine with that if you think it's easier